### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by quangdvn-sun**
Chapter 4 - Rails-flavored Ruby
Screen Shot 2020-07-24 at 2 27 47 PM

### CODE REVIEW

**Comment by quangdvn-sun** [comment]
ready

**Comment by hieplt-0914** [comment]
x

**Comment by quangdvn-sun** [comment]
ready

**Comment by quangdvn-sun** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
```
Comment (saobv-2223) [inline app/helpers/application_helper.rb:2]: Bỏ cmt

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %></title>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb:4]: BỎ ()

File: test/controllers/static_pages_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,36 @@
+require "test_helper"
```
Comment (saobv-2223) [inline test/controllers/static_pages_controller_test.rb:1]: BỎ test đi em

**Pull request merged**